### PR TITLE
added styles for badge

### PR DESCRIPTION
**Pull request by aishwarya-bm**
added styles for badge - text,button,icon,avatar badge variants
badge

### CODE REVIEW

**Review by shubhamkumar648 — COMMENTED** [review]
everything is good, you could  also use relative unit length, like rem instead of px, for better performance on web page

**Comment by aishwarya-bm** [comment]
> everything is good, you could also use relative unit length, like rem instead of px, for better performance on web page

Made changes,thanks!

**Review by sakleprajwal — COMMENTED** [review]
All good. Please go ahead

File: stylesheets/components/badge.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  line-height: 2rem;
+  height: 2rem;
+  font-size: 1.125rem;
+}
+
+.badge-left {
+  left: -0.75rem;
+}
+
+.position-absolute {
```
Comment (kedark152) [inline stylesheets/components/badge.css:32]: This generalized classes can be moved to base class or main class, as these can be used for other components.

**Review by kedark152 — COMMENTED** [review]
Looks Good!!

**Pull request merged**